cove_360/templates/cove_360/explore.html

```diff
@@ -26,7 +26,7 @@ <h3 class="panel-title panel-title-explore">
       <div id="data-supplied-body" class="collapse {% if conversion == 'flattenable' %}in{% endif %}">
         <div class="panel-body">
            <p class="explanation">
-	   {% trans "See the contents of this data at a glance. Use this to get an overview of the data and to check this was what you were expecting." %}
+	   {% trans "Use this section to get an overview of the data and check this was what you were expecting to see." %}
```

Comment:
Sorry @robredpath I didn't see this one before. This would be also good to be indented.